### DISCUSSION TITLE
Duplicated links and simplification of sentences

### DISCUSSION THREAD

**Discussion by IshitaTakeshi** [autowarefoundation/autoware_core_universe_prototype#202] [Q&A] [answered]
### 1. Duplicated links

I'm seeing the raw text of the [contribution guideline](https://autowarefoundation.github.io/autoware-documentation/main/contributing/) and it has two consecutive `#contribution-workflows`. Is this a mistake?

```
First, conform to the [Code of conduct](#contribution-workflows) section for the general manners in contributions.

See the [Contribution workflows](#contribution-workflows) section for the workflows.  
```

### 2. Sentence simplification

It also says 

```
It is recommended to read the Contributing pages in ROS 2 Docs as well because Autoware's guidelines generally depend on them.
```

but it implicitly asks contributors to know all the ROS2 contribution manners beforehand. We should decrease contributors' stress and burden so I think it's better to say

```
We follow the ROS2 contribution guide unless otherwise stated.
```

Then contributors can refer to the ROS2 contribution guide when they cannot find a proper guideline in our documentation.

### 3. Link to the OSS guide

In the last of the section, it says

```
For general information about OSS contributions, Open Source Guides by GitHub is a good resource.
```

Is this necessary? I think most of the contributors already know the basic instructions or they can search by themselves, and also this documentation contains general information about OSS so not very much related to this documentation.

**Comment by kenji-miyake** [comment]
> I'm seeing the raw text of the [contribution guideline](https://autowarefoundation.github.io/autoware-documentation/main/contributing/) and it has two consecutive #contribution-workflows. Is this a mistake?

Ah, yes. It seems a mistake.

**Comment by kenji-miyake** [comment]
> 2. Sentence simplification

@LalithVipulananthan What do you think about this phrase? I'd like to follow your opinion.

**Reply by ghost** [reply]
Apologies for the late response. 😓
I think that most of these points were addressed in my PR, but I will still comment as requested. 

Without context, the phrase doesn't make sense. It would be better to say something like "Simplify sentences to improve clarity" to be explicit about why a sentence should be simplified. I note that we should always aim to make the documentation clear and concise, but that simplification runs the risk of removing nuance that was intended by the writer. 

On the specific point about the ROS 2 guidelines, I believe that the ROS 2 contribution tenets were the things that we wanted Autoware contributors to keep in mind, so perhaps we should think about how to include our own form of those tenets in the Autoware contributing guide?

**Comment by kenji-miyake** [comment]
> 3. Link to the OSS guide

@LalithVipulananthan And could you give me some comments on this as well? :pray: 

I personally feel that supposing all contributors to know this is not good.

**Reply by ghost** [reply]
I agree. It is an assumption that new Autoware contributors are already familiar with OSS projects, and we should provide links to documentation that we think is relevant rather than making new contributors search for themselves.

**Comment by kenji-miyake** [comment]
@IshitaTakeshi If the issues have been resolved by https://github.com/autowarefoundation/autoware-documentation/pull/91, please mark this as an answer.

**Comment by IshitaTakeshi** [comment, marked as answer]
https://github.com/autowarefoundation/autoware-documentation/pull/91
